settings/nitrofiles/languages/english.ini

```diff
@@ -205,8 +205,8 @@ DESCRIPTION_LANGUAGE_1=Avoid the limited selections of your console language by
 DESCRIPTION_GAMELANGUAGE_1=Set language for DS ROMs played using nds-bootstrap.
 DESCRIPTION_TITLELANGUAGE_1=Set Banner Title language for DS ROMs (except DSiWare).
 DESCRIPTION_COLORMODE=Changes the color of your screens. May not work in all areas. Exit settings for the change to take effect.
-DESCRIPTION_SDREMOVALDETECTION=Enables/Disables SD removal detection. Disable it, if an error message is shown, instead of the ROM menu.
-DESCRIPTION_S1SDACCESS_1=Allows your flashcard to be used as a secondary device. Turn this off, if IR functionality doesn't work, or if the app crashes.
+DESCRIPTION_SDREMOVALDETECTION=Enables/Disables SD removal detection. Disable it if an error message is shown instead of the ROM menu.
+DESCRIPTION_S1SDACCESS_1=Allows your flashcard to be used as a secondary device. Turn this off if IR functionality doesn't work or if the app crashes.
 DESCRIPTION_WIFILED=Enables/Disables the DSi console's WiFi LED.
```

Comment:
Toggles the Nintendo DSi WiFi LED

Reply:
Hmm, is toggles really right for that? Its for an on / off selector while I feel toggles would be more for a button

Reply:
> Select whether to use the Nintendo DSi's WiFi LED.

I think that might be more fitting

Reply:
I think so, but if you want, we could make a poll on discord.

Reply:
> I think this might be more fitting: Select whether to use the Nintendo DSi's WiFi LED.

I don't own a DSi (for the next two weeks, that is), so I'll take your word on it. I'd assume nds-bootstrap uses it somehow? If so, then your way is better

Reply:
You set it to On or Off and it turns on / off the LED right there, so yours works... I just think that sounds kinda odd for an on off selection...